src/components/todo-list-edit-modal/TodoListEditModal.tsx

```diff
@@ -0,0 +1,188 @@
+import DatePicker from 'react-datepicker';
+import { Category, Color } from '@todo/enums';
+import { TodoItem } from '@todo/store';
+
+const newTodoItem: TodoItem = {
+  title: '',
+  description: '',
+  isImportant: false,
+  isCompleted: false,
+  createdDate: new Date().toDateString(),
+};
+
+type TodoItemProps = {
+  todoItem: TodoItem;
+  isEdit: false;
+  onClose: boolean;
+};
+
+const TodoListEditModal = ({ todoItem, isEdit, onClose }: TodoItemProps) => {
+  if (!isEdit) {
+    todoItem = newTodoItem;
+  }
```

Comment:
What is this? You can't change component props like this. Please read/learn the article: https://www.developerway.com/posts/react-re-renders-guide

Reply:
Removed